### PR TITLE
Migrate paper changes to whitepaper

### PR DESCRIPTION
**Pull request by adityasaky**

### CODE REVIEW

**Comment by adityasaky** [comment]
This is still a work in progress with more changes coming soon, but the stuff here should be good for review.

File: scudo-whitepaper.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ To start with, software supply chain attacks are on the rise. According to the [
 
 If there is any upside to this, it is that no one is ignoring supply chain issues anymore.  In the spring of 2021, President Joseph R. Biden issued a number of Executive Orders related to supply chains in the wake of this and other hacks. One of these executive orders, [#14028](https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/), specifically targets the development, testing, and distribution of software.  This represents a positive step forward and any actions resulting from these orders should reduce the risk to the software supply chain.
```
Comment (jhdalek55) [inline scudo-whitepaper.md]: In the spring of 2021, President Joseph R. Biden issued a number of Executive Orders related to supply chains in the wake of SolarWinds and other hacks.

In the edits we lost the reference to the "this" in "...this and other hacks"

File: scudo-whitepaper.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ To start with, software supply chain attacks are on the rise. According to the [
...
...les presents a particular challenge for several reasons. First off, an automobile has, on average, some [30,000 individual components](https://knowhow.napaonline.com/how-many-parts-are-in-a-car/), which can be sourced from perhaps hundreds of individual suppliers, and controlled within the vehicle by as many as [150 distinct electronic computing units (ECUs)](https://www.eenewsautomotive.com/en/number-of-automotive-ecus-continues-to-rise/). Thus, it can be viewed as an extreme example of the risks inherent in the use of third-party party software components mentioned above. The ECUs in cars are particularly diverse in suppliers, with one auto executive admitting in 2020 that only [10% of the software](https://spectrum.ieee.org/software-eating-car)  in its vehicles is developed in-house. With so many outside developers committing to the code base of a car, it is difficult to impose a consistent software supply chain protection strategy from company to company, or even from model to model. Similarly, the proprietary orientation of the auto industry presents another challenge. How can one protect a software supply chain without knowing the exact nature of the units you are protecting?
```
Comment (jhdalek55) [inline scudo-whitepaper.md]: Remove the repeated word "party" in the 14th line ("...an extreme example of the risks inherent in the use of third-party..." Take out the word "party" that follows.

**Review by jhdalek55 — CHANGES_REQUESTED** [review]
see two small fixes in text

**Review by jhdalek55 — APPROVED** [review]

**Review by jhdalek55 — APPROVED** [review]

File: scudo-whitepaper.md [hunk 3]
```diff
@@ -163,7 +163,9 @@ Once the Director repository decides which images are required and signs targets
...
+Performing full in-toto verification can be a resource intensive process for some vehicle ECUs, such as those designated as partial verification ECUs in Uptane. In these situations, other ECUs capable of performing in-toto verification are expected to step in for the the ECUs that cannot verify for themselves. This responsibility could fall to the primary ECU or a collection of full verification secondaries that share the extra load on behalf of partial verification secondary ECUs.
```
Comment (jhdalek55) [inline scudo-whitepaper.md]: Minor capitalizations for consistency...Primary ECU ...full verification Secondary, etc.

File: scudo-whitepaper.md [hunk 4]
```diff
@@ -163,7 +163,9 @@ Once the Director repository decides which images are required and signs targets
...
+
+For vehicles with no ECUs capable of performing in-toto verification, as a stopgap measure, the Image repository may perform in-toto verification to generate a new “summary link” metadata file. The link can be used as proof that the Image repository performed the complete verification workflow, and the repository can then send this summary link metadata to the vehicle. Using this lightweight verification workflow requires more trust in the Image repository, as it is performing the verification rather than the client, which in this case would be the primary ECU. Further, in these situations, the Image repository could become a single point of failure for thousands of vehicles. So, while this approach can significantly reduce the metadata and verification overhead, it must only be used when full verification is categorically impossible on the vehicle.
```
Comment (jhdalek55) [inline scudo-whitepaper.md]: Move "as a stopgap measure" to the front of the first sentence --- "As a stopgap measure for vehicles with no ECUs capable of performing in-toto verification, the Image repository may..." Also again Primary ECU

File: scudo-whitepaper.md [hunk 5]
```diff
@@ -163,7 +163,9 @@ Once the Director repository decides which images are required and signs targets
...
 
 Scudo can clearly combine the state of the art in software update security and software supply chain security systems. However, as with any framework that relies on user-defined policies, Scudo also requires that its policies be designed with secure defaults, allowing each part of the software supply chain as much privilege as is absolutely necessary.
```
Comment (jhdalek55) [inline scudo-whitepaper.md]: Add the word "only" to the last sentence  i.e. "allowing each part of the software supply chain only as much..."

**Review by jhdalek55 — CHANGES_REQUESTED** [review]

File: scudo-whitepaper.md (hunk 1 above)
Comment (adityasaky) [reply scudo-whitepaper.md]: Added reference to Sunburst in the previous line, upside to... because we'd have the same issue there otherwise.

**Comment by adityasaky** [comment]
Resolved your comments I believe, @jhdalek55!

**Review by jhdalek55 — APPROVED** [review]

**Comment by jhdalek55** [comment]
Ok. These are good to go as far as I'm concerned.

**Comment by adityasaky** [comment]
cc @mnm678 Do we publish an updated version of this and the PDF?

**Review by mnm678 — APPROVED** [review]

**Comment by mnm678** [comment]
> cc @mnm678 Do we publish an updated version of this and the PDF?

Yes, this looks good to me

**Comment by adityasaky** [comment]
I don't have merge powers on this repo. I think we can go ahead though. :)

**Comment by jhdalek55** [comment]
Done. Thanks.

**Pull request merged**